tests/conftest.py

```diff
@@ -55,6 +55,11 @@ def pytest_addoption(parser):
         'not within the clouds to be run, it will be reset to the first '
         'cloud in the list of the clouds to be run.')
 
+    parser.addoption('--terminate-on-failure',
```

Comment:
Comment for ~L16: we should break paras to improve readability (see styleguide)

```
# By default, only run generic tests and cloud-specific tests for GCP and Azure,
# due to the cloud credit limit for the development account.
# 
# A "generic test" tests a generic functionality (e.g., autostop) that
# should work on any cloud we support. The cloud used for such a test
# is controlled by `--generic-cloud` (typically you do not need to set it).
#
# To only run tests for a specific cloud (as well as generic tests), use
# --aws, --gcp, --azure, or --lambda.
#
# To only run tests for managed spot (without generic tests), use --managed-spot.
```

Reply:
Done. Thanks!